src/Sulu/Component/Content/Document/Subscriber/BlameSubscriber.php

```diff
@@ -147,11 +147,9 @@ public function setBlamesOnNode(
 
         $creatorPropertyName = $this->propertyEncoder->encode($encoding, static::CREATOR, $locale);
         if (!$node->hasProperty($creatorPropertyName)) {
-            $accessor->set(self::CREATOR, $userId);
-            $node->setProperty(
-                $creatorPropertyName,
-                $document->getCreator()
-            );
+            $creator = $document->getCreator() ?: $userId;
```

Comment:
we want to use the creator from the document, if it is set. we already use the same strategy in the `TimestampSubscriber`:
https://github.com/sulu/sulu/blob/1af4ee3cc12dd9ecca2a39f90ec57d136326e677/src/Sulu/Component/DocumentManager/Subscriber/Behavior/Audit/TimestampSubscriber.php#L150-L154